releases.md

```diff
@@ -8,7 +8,7 @@ versions, as well as to the latest version and the nightly build of
 documentation. Each set of versioned docs includes a Release Nodes
 page for that particular version.
 {%- if site.archive %}
-- [{{site.data.versions.first.title}} (latest)](/)
+- [{{site.data.versions.first.title}} (latest)](/) [(permalink)](/archive/{{site.data.versions.first.title}})
```

Comment:
`title` has the full version including the revision (vX.Y.Z). You just want the major/minor (vX.Y) which is in `page.version`.
```suggestion
- [{{site.data.versions.first.title}} (latest)](/) [(permalink)](/archive/{{page.version}})
```